### PR TITLE
Create AngularJS Component for the VWS

### PR DESCRIPTION
**Pull request by favreau**
Added the AngularJS component as an app example, and made it available in debug mode via Django static files features.

### CODE REVIEW

File: rendering_resource_manager_service/service/settings.py [hunk 1]
```diff
@@ -45,7 +45,7 @@
 API_VERSION = 'v1'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```
Comment (dnachbaur) [inline rendering_resource_manager_service/service/settings.py:48]: keep that?

**Comment by dnachbaur** [comment]
+1 otherwise

**Comment by bilgili** [comment]
Change log update ?

File: rendering_resource_manager_service/service/settings.py (hunk 1 above)
Comment (favreau) [inline rendering_resource_manager_service/service/settings.py:48]: Yes, static files are only served when DEBUG is activated. The documentation specifies that Django should NOT be used for production anyway.

**Comment by bilgili** [comment]
+1

**Pull request merged**